### DISCUSSION TITLE
A few questions about sveltekit and sveltia cms

### DISCUSSION THREAD

**Discussion by NVCDevelopmentTeam** [sveltia/sveltia-cms#283] [Q&A]
Hello, I want to directly integrate sveltia cms into my sveltekit project, what should I do? There needs to be a file in the project's root directory, right?

**Comment by kyoshino** [comment]
I cannot help with setup for a specific framework, but see #250 for basic info on SvelteKit.

**Comment by NVCDevelopmentTeam** [comment]
I have read and configured the way to use scripts, however, that way the website loads very slowly when deployed to vercel and I used my method which is that I installed a package via npm and then created an API Endpoint for it. works and it integrates quite well into sveltekit but one thing is that it still loads faster than attaching a script but is slow compared to the website and not very optimized. So I created a configuration file called sveltia.config.js in the root directory of the project and pulled the data from there. However, it has an error and cannot load data like a regular library like mdsvex. You can review my code if I did anything wrong: https://github.com/NVCDevelopmentTeam/Zepp-OS-ScreenReader.git

**Comment by NVCDevelopmentTeam** [comment]
I saw you mention that in addition to using .yml java script, you can also use .json files. However, one thing I wonder about is that when running on localhost, it loads very quickly, but when deploying to vercel, the cms cannot load the config file. So, is my configuration correct?
